Pinta.Tools/Tools/TextTool.cs

```diff
@@ -413,6 +417,11 @@ private void HandleSelectedLayerChanged (object? sender, EventArgs e)
 		UpdateFont ();
 	}
 
+	protected override void OnChangeAntialias (object? sender, EventArgs e)
+	{
+		UpdateFont();
```

Comment:
I noticed that after I finished typing some text (i.e. after pressing `Esc`) that the antialiasing changes didn't cause a redraw right away until I did something else to force a redraw, like pressing Ctrl.

This also happens with other font properties too, though, so it's a more general behaviour and could probably be left for another pull request

Reply:
Yes, it seems like in that state after pressing escape, the text is not considered in edit mode, and so does not call redraw.

That edit flag seems to be there for a reason (starting without it with text tool on causes crash), so i won't touch it for the time being in case it causes other issues.